core/quality/morqa_transport.go

```diff
@@ -36,17 +36,17 @@ type morqaTransport struct {
 }
 
 func (transport *morqaTransport) SendEvent(event Event) error {
-	if metric := mapEventToMetric(event); metric != nil {
-		return transport.morqaClient.SendMetric(metric)
+	if id, metric := mapEventToMetric(event); metric != nil {
+		return transport.morqaClient.SendMetric(id, metric)
 	}
 
 	return fmt.Errorf("event not implemented")
 }
 
-func mapEventToMetric(event Event) *metrics.Event {
+func mapEventToMetric(event Event) (string, *metrics.Event) {
 	switch event.EventName {
-	case startupEventName:
-		return nodeVersionToMetricsEvent(event.Application)
+	case unlockEventName:
```

Comment:
Not sure how useful is unlock event.

Reply:
This is the first place where we are using Identity on application start.